net-misc/axel/axel-2.16.1.recipe

```diff
@@ -7,32 +7,36 @@ HOMEPAGE="https://github.com/axel-download-accelerator/axel/"
 COPYRIGHT="2001-2007 Wilmer van der Gaast,
 	2007-2009 Giridhar Appaji Nag,
 	2008-2010 Philipp Hagemeister,
-	2015-2016 Joao Eriberto Mota Filho,
-	2016 Stephen Thirlwall, and others."
+	2015-2017 Joao Eriberto Mota Filho,
+	2016-2017 Stephen Thirlwall,
+	2017 Ismael Luceno,
+	2017 Antonio Quartulli, and others."
 LICENSE="GNU GPL v2"
 REVISION="1"
 SOURCE_URI="https://github.com/axel-download-accelerator/axel/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="0e223f18954e4c6c34b882a474c526b9c7d107168220c2f3892598248236a172"
+CHECKSUM_SHA256="64529add74df3db828f704b42d4ec3fcdacb8142c84f051f9213637c337e706c"
+SOURCE_FILENAME="axel-$portVersion.tar.gz"
 PATCHES="axel-$portVersion.patchset"
 
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	axel = $portVersion
-	cmd:axel = $portVersion
+	cmd:axel$secondaryArchSuffix = $portVersion
 	"
 REQUIRES="
-	haiku
-	lib:libcrypto
-	lib:libintl
-	lib:libssl
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
-	haiku_devel
-	devel:libcrypto
-	devel:libiconv
-	devel:libgettextlib
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libgettextlib$s$secondaryArchSuffix
 	devel:libssl
```

Comment:
```diff
-	devel:libgettextlib$s$secondaryArchSuffix
+	devel:libgettextlib$secondaryArchSuffix
-  	devel:libssl
+  	devel:libssl$secondaryArchSuffix
```
(BTW, I think you did not notice [this comment](https://github.com/haikuports/haikuports/pull/2636/commits/30db8f464017f9c3dfe2d757557e415e4ecf7823#r192557498).)
(I did not test the recipe but it looks like everything else is ok.)